releases/release-6.1.3.md

```diff
@@ -0,0 +1,92 @@
+
+    - (dup) Fix the issue that the `grantor` field is missing in the `mysql.tables_priv` table [#38293](https://github.com/pingcap/tidb/issues/38293) @[CbcWestwolf](https://github.com/CbcWestwolf)
+    - (dup) Fix the issue of the wrong query result that occurs when the mistakenly pushed-down conditions are discarded by Join Reorder [#38736](https://github.com/pingcap/tidb/issues/38736) @[winoros](https://github.com/winoros)
+    - Fix an issue that `get_lock()` cannot hold for more than 10 minutes. [#38706](https://github.com/pingcap/tidb/issues/38706) @[tangenta](https://github.com/tangenta)
+    - Fix the issue that the auto increment column cannot be used with check constraint. [#38894](https://github.com/pingcap/tidb/issues/38894) @[YangKeao](https://github.com/YangKeao)
+    - Fix broken log rotation of grpc [#38941](https://github.com/pingcap/tidb/issues/38941) @[xhebox](https://github.com/xhebox)
+    - Delete TiFlash sync status from etcd when table is truncated or dropped [#37168](https://github.com/pingcap/tidb/issues/37168) @[CalvinNeo](https://github.com/CalvinNeo)
+    - Fix the issue of arbitrary file read via data source name injection (CVE-2022-3023). [#38541](https://github.com/pingcap/tidb/issues/38541) @[lance6716](https://github.com/lance6716)
+
+    <!--executor owner: zanmato1984-->
+
+    <!--planner owner: qw4990-->
+
+    <!--transaction owner: cfzjywxk-->
+
+    - (dup) Fix the issue that in some scenarios the pessimistic lock is incorrectly added to the non-unique secondary index [#36235](https://github.com/pingcap/tidb/issues/36235) @[ekexium](https://github.com/ekexium)
+
+- PD
+
+    <!--owner: nolouch -->
+
+    - (dup) Fix inaccurate Stream timeout and accelerate leader switchover [#5207](https://github.com/tikv/pd/issues/5207) @[CabinfeverB](https://github.com/CabinfeverB)
+
++ TiKV
+
+    <!--owner: tonyxuqqi-->
+
+    - (dup) Fix abnormal Region competition caused by expired lease during snapshot acquisition [#13553](https://github.com/tikv/tikv/issues/13553) @[SpadeA-Tang](https://github.com/SpadeA-Tang)
+
++ TiFlash
+
+    <!--compute owner: zanmato1984 -->
+
+    - (dup) Fix the issue that logical operators return wrong results when the argument type is UInt8 [#6127](https://github.com/pingcap/tiflash/issues/6127)
+    - (dup) Fix the issue that wrong data input for `CAST(value AS DATETIME)` causing high TiFlash sys CPU [#5097](https://github.com/pingcap/tiflash/issues/5097) @[xzhangxian1008](https://github.com/xzhangxian1008)
+
+    <!--storage owner: flowbehappy -->
+
+    - Fix the issue that heavy write pressure could generate too many column files in delta layer. [#6361](https://github.com/pingcap/tiflash/issues/6361) @[lidezhu](https://github.com/lidezhu)
+    - Fix the issue that column files in delta layer cannot be compacted after restart tiflash [#6159](https://github.com/pingcap/tiflash/issues/6159) @[lidezhu](https://github.com/lidezhu)
+
++ Tools
+
+    + Backup & Restore (BR)
+
```

Comment:
Fix the issue that fail to restore when use old collation framework in databases or tables.  [#39150](https://github.com/pingcap/tidb/issues/39150)